cmd/up/activate.go

```diff
@@ -340,3 +341,9 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context)
 		}
 	}
 }
+
+func getPullingMessage(message string) string {
+	var oktetoRegistryRegex = regexp.MustCompile(`registry\..*\.okteto\.net\/.*\/`)
```

Comment:
https://github.com/okteto/okteto/blob/master/pkg/okteto/auth.go#L230 gives you the url of the registry, and `up.Dev.Namesapace` gives you the current namespace. If you append both values it gives you what should be replaced by `okteto.dev`